vud.theme.inc

```diff
@@ -121,7 +144,7 @@ function vud_pseudo_theming($vote_entity, $template_type, $plugin, &$variables)
     // @fixme: drupal_discover_template() was removed from core.
     //$template_file = drupal_find_theme_templates($paths, $suggestions, $plugin['extension']);
   }
-
+  
```

Comment:
Spaces without a real line are against coding standards, please remove this unneeded and wrong change.